beam/www/beam/index.ts

```diff
@@ -0,0 +1,118 @@
+import { createApp } from 'vue'
+import { createMemoryHistory, createRouter } from 'vue-router'
```

Comment:
I don't think the Memory History  API is the right approach.

Reply:
@bhattdevarsh based on #138, let's switch this one to hash mode.